sphinx_gallery/tests/test_scrapers.py

```diff
@@ -80,7 +80,7 @@ def test_save_matplotlib_figures(gallery_conf, ext):
 def test_save_matplotlib_figures_hidpi(gallery_conf):
     """Test matplotlib hidpi figure save."""
     ext = "png"
-    gallery_conf["image_srcset"] = ["2x"]
+    gallery_conf["image_srcset"] = [2]
```

Comment:
Sorry, I'm confused, 'image_srcset' does not have to be a string with 'x' at the end?

Reply:
It does, but now it's normalized in the configuration initialization, to avoid doing that on every scrape. But these tests appear to skip that step. I could add that here (and actually maybe that should be tested as well.)

Reply:
Ahh right, that is confusing. 

> I could add that initialization here instead

+1

> actually maybe that should be tested as well.)

+1

Reply:
Done, though this spilled over into some other tests. Not sure if that warrants a separate PR.